plopfile.js

```diff
@@ -50,13 +50,11 @@ module.exports = function (plop) {
         const indexFile = `${plop.getDestBasePath()}/packages/components/src/index.ts`;
 
         if (fs.existsSync(indexFile)) {
-          const nameRegex = /default as ([^\s]*)/;
```

Comment:
So I think plop is broken on `main` (but haven't tested that). Likely broke when we started exporting types, as I don't think this regex handles that.

Not related to the updates, but I noticed this when trying to update Plop.